app/assets/stylesheets/darkswarm/base/colors.css.scss

```diff
@@ -1,4 +1,13 @@
+@import '../branding';
+
 $modal-background-color: #efefef;
 $modal-content-background-color: #fff;
 $modal-alert-link-color: #fff;
 $modal-alert-link-hover-color: rgba(255, 255, 255, .7);
+
+$cookies-banner-background-color: $dark-grey;
```

Comment:
I'm also lost with CSS color vars naming but, does this mean we'll have a var for each of `$dark-grey` uses we'll have? won't we end up with trillions of single-purpose vars?

Reply:
@sauloperez Ideally, we'd keep in mind to extract CSS components when possible. For example, there could be just one set of color assignments in one SCSS partial for general styles used in all modals. So maybe not trillions :wink: but a hundred or so variables.

For the cookies banner though, the color assignments are quite unique, so they would be single-purpose.

What do you think?

Reply:
you mean having an SCSS file for each new UI component where we assign the palette vars to component vars? looks good :+1: naming colors vars seems like a very difficult problem for me. I'm struggling with my side project :sweat:

Reply:
To clarify, the style I suggested, which is also how @luisramos0 did it in this PR, is to _assign_ the palette vars to component vars in a central file. Each component SCSS partial only _uses_ the component vars.

Good luck with your side project! :worried:

Reply:
:+1:
yes, that central file is this colors.scss file.
if it gets to big, we can move some parts to separate files, as you say, for example, modals_colors.scss

The new convention here is that when you set a color in your component scss, you always set it to a variable like in menu.scss:
 li a { color: $menu-mobile-li-a-color; }
instead of 
 li a { color: rgba(0, 0, 0, 0.9); }
or
 li a { color: $dark-grey; }

and then you go to colors.scss to assign this variable $menu-mobile-li-a-color = $dark-grey;

Reply:
Let's doc this if the rest of the team agrees.